src/content/blog/astro-tutorial.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Astro Tutorial
+description: 'Wrapping up Astro Docs week with our biggest Docs announcement to date: our first Astro Tutorial.'
+publishDate: 'November 4, 2022'
+socialImage: '/assets/blog/astro-tutorial/cover.png'
+coverImage: '/assets/blog/astro-tutorial/cover.png'
+lang: 'en'
+authors:
+    - chris
+    - sarah
+    - dan
+    
+---
+
+import BlogContentImage from '/src/components/BlogContentImage.astro'
+import community from '/src/assets/blog/docs-hacktoberfest/contributors.jpg'
+
+We hope you enjoyed Astro Docs Week, where our Docs core maintainers took control of the keyboard and our Twitter account! Each day, we posted a peek behind the scenes of life on Team Docs:
+
+- our [low-code/no-code Hacktoberfest drive](https://twitter.com/astrodotbuild/status/1587160080717434881)
```

Comment:
I guess this doesn’t really need to include both these `-code` adjectives — probably cleaner like this:

```suggestion
- our [low-code Hacktoberfest drive](https://twitter.com/astrodotbuild/status/1587160080717434881)
```